sdk/tests/js/tests/compressed-tex-image.js

```diff
@@ -97,8 +97,10 @@ if (!gl) {
       wtu.shouldGenerateGLError(gl, gl.NO_ERROR,
           `gl.compressedTexImage2D(gl.TEXTURE_2D, 0, ext.${enumName}, ${blockSize},${blockSize}, 0, g_view)`);
 
-      wtu.shouldGenerateGLError(gl, gl.NO_ERROR,
-          `gl.compressedTexSubImage2D(gl.TEXTURE_2D, 0, 0,0, ${blockSize},${blockSize}, ext.${enumName}, g_view)`);
+      if (supportsSubImage) {
+        wtu.shouldGenerateGLError(gl, gl.NO_ERROR,
+            `gl.compressedTexSubImage2D(gl.TEXTURE_2D, 0, 0,0, ${blockSize},${blockSize}, ext.${enumName}, g_view)`);
+      }
```

Comment:
Please add in the negative test - that this should generate INVALID_OPERATION per https://www.khronos.org/registry/OpenGL/extensions/OES/OES_compressed_ETC1_RGB8_texture.txt .

Reply:
Also - do you think you could update the associated tests in the conformance-suites/1.0.3 and conformance-suites/2.0.0 snapshots? Those need to stay valid before and after this validation change - so essentially this current change is the one that should be applied to those older snapshots.

Thanks in advance for your help!

Reply:
Unless I'm reading this wrong, the same compressed-tex-image.js file is used for both test suites.

Older versions of the test only used compressedTexImage2D, so the compressedTexSubImage2D issue wasn't exposed.

Reply:
> Thanks @jdashg for your review!
> 
> @sugoi1 still hoping for your feedback on backporting this change to the snapshots.

Hi Ken, as I mentioned, I don't see anything that needs to be backported to the snapshots. Only compressed-tex-image.js contains code that attempts to call compressedTexSubImage2D in this manner. If I'm wrong, point me to the file(s) and I'll do it.

Reply:
Ah, sorry - I missed that comment above. Thanks for checking and confirming the old test suites don't need to be updated.